tests/app/main/views/test_admin_manager.py

```diff
@@ -110,6 +117,25 @@ def test_should_list_alphabetically_with_all_suspended_users_below_active_users(
         assert "Suspended" in rows[5].text_content()
         assert "Has-been Sourcing Support" in rows[5].text_content()
 
+    def test_should_link_to_edit_admin_user_page(self, data_api_client):
+        self.user_role = "admin-manager"
+        data_api_client.find_users_iter.side_effect = [
+            iter(self.SUPPORT_USERS),
+            iter(self.CATEGORY_USERS),
+            iter(self.SOURCING_USERS)
+        ]
+        response = self.client.get("/admin/admin-users")
+        document = html.fromstring(response.get_data(as_text=True))
+
+        links = document.xpath("//td[@class='summary-item-field-with-action']//a")
+
+        assert links[0].values()[0] == "/admin/admin-users/9089/edit"
+        assert links[1].values()[0] == "/admin/admin-users/9088/edit"
+        assert links[2].values()[0] == "/admin/admin-users/9087/edit"
+        assert links[3].values()[0] == "/admin/admin-users/9092/edit"
+        assert links[4].values()[0] == "/admin/admin-users/9090/edit"
+        assert links[5].values()[0] == "/admin/admin-users/9091/edit"
```

Comment:
With a bit of xpath magic this could be rewritten as:

```
        links = document.xpath("//td[@class='summary-item-field-with-action']//a/@href")

        assert links == [
            "/admin/admin-users/9089/edit",
            "/admin/admin-users/9088/edit",
            "/admin/admin-users/9087/edit",
            "/admin/admin-users/9092/edit",
            "/admin/admin-users/9090/edit",
            "/admin/admin-users/9091/edit"
        ]
```

Reply:
Ahh, neat 💖